Gemfile

```diff
@@ -36,7 +36,7 @@ gem 'validates_email_format_of', '~> 1.6', '>= 1.6.3'
 gem "geocoder"
 # Rails 4 support is a work in progress so requires tracking master
 gem 'activeadmin', '~> 1.0.0.pre2'
-gem "devise"
+gem "devise", '~> 3.5'
```

Comment:
Why are we specifying the version of devise? I find it handy to say so in a comment so that down the track you understand the rationale.